specification/devops/resource-manager/readme.csharp.md

```diff
@@ -0,0 +1,15 @@
+## C
+
+These settings apply only when `--csharp` is specified on the command line.
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+```yaml $(csharp)
+csharp:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 1
+  clear-output-folder: true
+  client-side-validation: false
+  namespace: Microsoft.DevOps
```

Comment:
prefer to add prefix for namespace: Microsoft.Azure.
For namespaces and output-folder of other languages, please refer to files in https://github.com/Azure/azure-rest-api-specs/tree/master/specification/cdn/resource-manager